CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 IMPROVEMENTS:
 * Control Plane
   * Upgrade Docker image Alpine version from 3.13 to 3.14. [[GH-737](https://github.com/hashicorp/consul-k8s/pull/737)]
-
+* Helm Chart
+  * Enable injecting sidecar container via helm configuration.  [[GH-749](https://github.com/hashicorp/consul-k8s/pull/749)]
```

Comment:
```suggestion
  * Enable adding extra containers to server and client Pods.  [[GH-749](https://github.com/hashicorp/consul-k8s/pull/749)]

```

Reply:
Just some nits.

* we already use "injection" to mean automatically adding sidecars to service mesh pods so I think to differentiate we can use "adding extra containers" here instead. It also matches up with the value.
* It's already under Helm chart so probably don't need the "via helm config".
* Added a newline to separate from the `## 0.34..`